src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -260,7 +260,7 @@ void GraphOptimizer::FuseConvMatmulFCDeconvAndDQScales(Graph &graph) {
         auto scalesDims = getNormalizedDimsBySize(scales->getOutputShapeAtPort(0).getDims(),
                                                 node->getOutputShapeAtPort(0).getDims().size());
         auto scaleSize = std::accumulate(scalesDims.begin(), scalesDims.end(), 1, std::multiplies<size_t>());
-        node->initializeDQScales(scalesData, scaleSize);
+        node->setDQScales(scalesData, scaleSize);
```

Comment:
So `initializeDeQuantizedScales` will be also called twice in INT8 tiny-SWIN model on pattern of FC + Multiply-per channel + Multiply DQ multiply? the name `initialize` seems to imply that it's only called once for each node, maybe change it to `append` or `fuse`?

Reply:
rename to fuseDequantizedScale.